Munis

muniseben opened the pull request:
check check check

muniseben committed:
treeForm component created

muniseben committed:
alignment added on tree page

muniseben committed:
picture section added

NadegeHbg commented:
You should change the class to className and the for to htmlFor to avoid error in console

muniseben committed:
for changed to forHTML
class changed to className
axious added
cors added
added test_server.js for testing
express added

muniseben committed:
master branch accepted
munis branch changes added

muniseben committed:
Added design section on legalPolicy page
Added design section on contact page

muniseben committed:
comment added
